app/MMB/src/scenes/tripServices/insurance/InsuranceTermsPdfScene.js

```diff
@@ -0,0 +1,42 @@
+// @flow strict
+
+import * as React from 'react';
+import idx from 'idx';
+import { PrivateApiRenderer, graphql } from '@kiwicom/mobile-relay';
+import PdfViewer from '@kiwicom/mobile-pdf';
+
+import type { InsuranceTermsPdfSceneQueryResponse as Props } from './__generated__/InsuranceTermsPdfSceneQuery.graphql';
+
+const renderPdfViewer = (props: Props) => {
+  const documents = idx(props, _ => _.allDocuments.edges) || [];
+  let uri = '';
+  const document = idx(documents, _ => _[0].node);
+  if (document && document.__typename === 'InsuranceTerms') {
+    uri = document.url || '';
+  }
+  return <PdfViewer uri={uri} />;
```

Comment:
also I am a little bit unsure about your approach. If someone adds another document type, it is not sure insurance document will be the first one returned anymore

you should probably filter the list until you find typename InsuranceTerms

Reply:
But since I use 
```
node {
                ... on InsuranceTerms {
                  url
                  __typename
                }
              }
```
there should only be documents of `InsuranceTerms` type in the response. And the version corresponding to the language is chosen by the server too. Maybe I should make the response less smart and let the app decide what it wants to do with the different versions in different languages?